docs/attributes-registry/gen-ai.md

```diff
@@ -30,6 +30,13 @@ This document defines the attributes used to describe telemetry in the context o
 
 `gen_ai.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
 
-| Value    | Description | Stability                                                        |
-| -------- | ----------- | ---------------------------------------------------------------- |
-| `openai` | OpenAI      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| Value         | Description  | Stability                                                        |
+| ------------- | ------------ | ---------------------------------------------------------------- |
+| `openai`      | OpenAI       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `microsoft`   | Microsoft    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `huggingface` | Hugging Face | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `google`      | Google       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `meta`        | Meta         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
We should have a specific produce name rather than company here. 
Is there a common name for Meta AI? Llama?